src/main/groovy/org/mockito/release/gradle/ContributorsPlugin.java

```diff
@@ -0,0 +1,7 @@
+package org.mockito.release.gradle;
+
+import org.gradle.api.Plugin;
+import org.gradle.api.Project;
+
+public interface ContributorsPlugin extends Plugin<Project> {
```

Comment:
Feel free to drop the interface. I thought that the interfaces are a good idea but they are more of a hindrance during development.